bundle-size-chart/package.json

```diff
@@ -11,15 +11,12 @@
   },
   "main": "dist/app.js",
   "scripts": {
-    "lint": "eslint src --ext .js,.ts",
-    "fix": "npm run -- --fix",
     "dev": "nodemon",
     "deploy": "gcloud --project amp-bundle-size-chart app deploy",
     "deploy-cron": "gcloud --project amp-bundle-size-chart app deploy cron.yaml",
     "deploy-tag": "git tag 'deploy-bundle-size-chart-'`date -u '+%Y%m%d%H%M%S'`",
     "prestart": "tsc",
     "start": "node dist/app.js",
-    "pretest": "npm run lint",
```

Comment:
@rsimha IIRC All of the apps relied on `pretest` so that PRs touching that app would trigger those build targets, and each test target would perform a lint check as part of the testing pass (`npm pretest` is run automatically when `npm test` is run). I believe the result of this PR is that linting is now disabled in CI for this repo. Am I missing something?

Reply:
In addition to removing per-app linting, this PR added a new root-level script that runs lint across the entire repo:

https://github.com/ampproject/amp-github-apps/blob/dd5d251e2b31df7bb6264c669cb3f6104862c6d0/package.json#L13-L15

During CI, the check is run no matter which apps are touched by the incoming PR.

https://github.com/ampproject/amp-github-apps/blob/dd5d251e2b31df7bb6264c669cb3f6104862c6d0/build-system/pr-check.js#L37-L44
https://github.com/ampproject/amp-github-apps/blob/dd5d251e2b31df7bb6264c669cb3f6104862c6d0/build-system/pr-check.js#L57-L58

For example, see the root-level testing step in [this build](https://github.com/ampproject/amp-github-apps/runs/2758457826?check_suite_focus=true#step:4:8).

Linting the entire repo like this takes <20s, and saves significant noise / duplication when it comes to package management / upgrades.

Good question to have asked though. 👍

Reply:
It seems confusing that the root-level `npm run test` task exclusively lints and runs no tests. Perhaps `npm run test` could trigger tests for all apps, while the `pretest target` could lint and the CI config would call `npm run lint` directly since it's manually executing each test target. Currently, the CI collapses the output so all that's visible is `npm run test` so the presence of linting is hidden

Reply:
Fair point. We could change the root-level execution to `npm run lint` instead of hiding it behind `npm run test` to make things clear.